Kernel/FileSystem/SysFS/Subsystems/Kernel/GlobalInformation.cpp

```diff
@@ -51,7 +52,13 @@ ErrorOr<void> SysFSGlobalInformation::refresh_data(OpenFileDescription& descript
             return ENOMEM;
     }
     auto builder = TRY(KBufferBuilder::try_create());
-    TRY(const_cast<SysFSGlobalInformation&>(*this).try_generate(builder));
+    TRY(Process::current().jail().with([&](auto& my_jail) -> ErrorOr<void> {
+        // Note: If we are in a jail, don't let the current process to change the power state.
```

Comment:
Is this comment correct? This is the global information node, not the power state node.